unity/Holo/Assets/Scripts/model_with_plate/ModelWithPlate.cs

```diff
@@ -551,6 +551,17 @@ private ModelLayer ButtonOfLayer(GameObject buttonGameObject)
      */
     private void LoadInstance(string newInstanceBundleName, bool newIsPreview)
     {
+        if(instanceBundle != null)
+        {
+            Debug.Log("LoadInstance - currentInstanceName: " + instanceBundle.Name + ", newInstanceName: " + newInstanceBundleName);
+            if(instanceIsPreview != newIsPreview && instanceBundle.Layers.All(l => l.DataType == DataType.Volumetric))
+            {
+                Debug.Log("LoadInstance - preview accepted!");
+                instanceIsPreview = newIsPreview;
+                return;
+            }
+        }            
+
         UnloadInstance();
 
         instanceLoaded = true;

```

Comment:
Should it skip UnloadInstance() if it's volumetric?

Reply:
Yes, this is done on purpose.